### ISSUE TITLE
File/folder transfer between machines?

### ISSUE THREAD

**Issue by nickdandakis** [jlevy/the-art-of-command-line#15]
There seems to be no mention of scp or rsync, which I use on a fairly regular basis. Would it make sense to add it in here?

**Comment by jlevy** [comment]
Yeah, there should be. Feel free to make a PR if you'd like; otherwise I'll try to resolve a handful of these issues soon.

**Comment by uggla** [comment]
Hello,

I suggest to speak about -S option to rsync. This is to handle sparse file.
As an example, it can be used to copy correctly Oracle undo files.
By the way, what do you think about a couple of lines explaining sparse files ?

**Comment by mkwardakov** [comment]
There are tons of options in rsync that could be useful someday. We cannot mention them all.
